vm/devices/net/mana_driver/src/mana.rs

```diff
@@ -535,9 +538,10 @@ impl<T: DeviceBacking> Vport<T> {
         vport_link_status[vport_index] = LinkStatus::Active { sender, connected };
     }
 
-    /// Returns an object that can allocate host memory to be shared with the device.
-    pub async fn host_allocator(&self) -> DmaAllocator<T> {
-        self.inner.gdma.lock().await.device().host_allocator()
+    /// Returns an object that can allocate dma memory to be shared with the device.
+    pub async fn get_dma_client(&self) -> anyhow::Result<Arc<dyn DmaClient>>
```

Comment:
prefer not using `get_<thing>` naming in Rust, generally just call it by it's name, so in this case `dma_client` like how previously it was `host_dma_allocator`.